x-pack/plugins/ml/server/lib/alerts/alerting_service.ts

```diff
@@ -498,7 +498,7 @@ export function alertingServiceProvider(
         job_id: [...new Set(requestedAnomalies.map((h) => h._source.job_id))][0],
         is_interim: requestedAnomalies.some((h) => h._source.is_interim),
         anomaly_timestamp: timestamp,
-        anomaly_score: topAnomaly._source[getScoreFields(resultType, useInitialScore)],
+        anomaly_score: [topAnomaly._source[getScoreFields(resultType, useInitialScore)]],
```

Comment:
Unrelated to this PR, but noticed a typo in this file in the `
No anomalies have been found in the concecutive bucket after the alert was triggered.` message. Can you change `concecutive` to `consecutive`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/174919/commits/ab4fdbcc9b73514510d015c92b2247423218c9fb